docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -42,38 +40,22 @@ Download the following configuration files:
 
 {% capture lessoncontent %} 
 
-## Create a PersistentVolume
-
-MySQL and Wordpress each use a PersistentVolume to store data. While Kubernetes supports many different [types of PersistentVolumes](/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes), this tutorial covers [hostPath](/docs/concepts/storage/volumes/#hostpath).
+## Create PersistentVolumeClaims and PersistentVolumes
 
-**Note:** If you have a Kubernetes cluster running on Google Kubernetes Engine, please follow [this guide](https://cloud.google.com/kubernetes-engine/docs/tutorials/persistent-disk).
-{: .note}
+MySQL and Wordpress each require a PersistentVolume to store data.  Their PersistentVolumeClaims will be created at the deployment step.
 
-### Setting up a hostPath Volume
+Many cluster environments have a default StorageClass installed.  When a StorageClass is not specified in the PersistentVolumeClaim, the cluster's default StorageClass is used instead.
 
-A `hostPath` mounts a file or directory from the host node’s filesystem into your Pod. 
+When a PersistentVolumeClaim is created, a PersistentVolume is dynamically provisioned based on the StorageClass configuration.
 
-**Warning:** Only use `hostPath` for developing and testing. With hostPath, your data lives on the node the Pod is scheduled onto and does not move between nodes. If a Pod dies and gets scheduled to another node in the cluster, the data is lost. 
+**Warning:** In local clusters, the default StorageClass uses the `hostPath` provisioner.  `hostPath` volumes are only suitable for development and testing. With `hostPath` volumes, your data lives on the node the Pod is scheduled onto and does not move between nodes. If a Pod dies and gets scheduled to another node in the cluster, the data is lost.
```

Comment:
What's the default `hostPath` path? It may be a problem if the node filesystem is partially write-protected, but including the path here is good enough IMO.

Reply:
/tmp/hostpath_pv/<uuid>

hostpath provisioner is only for dev/local clusters, so I don't think we need to worry about node OSes here.  If you run this tutorial in GCE, it will use the gce-pd provisioner, and not the hostpath provisioner.

I'll add a note here that the provisioner will create a directory under /tmp, so the data is definitely not in a permanent location.